components/freertos/include/freertos/ringbuf.h

```diff
@@ -297,6 +297,18 @@ BaseType_t xRingbufferAddToQueueSetRead(RingbufHandle_t ringbuf, QueueSetHandle_
  */
 BaseType_t xRingbufferAddToQueueSetWrite(RingbufHandle_t ringbuf, QueueSetHandle_t xQueueSet);
 
+/**
+ * @brief Check if semaphore belongs to ringbuffer
```

Comment:
``@brief Check if the selected queue set member is the ringbuffer's read semaphore``